docs/docs/how-tos/disable-streaming.ipynb

```diff
@@ -113,9 +113,12 @@
    ],
    "source": [
     "input = {\"messages\": {\"role\":\"user\", \"content\":\"how many r's are in strawberry?\"}}\n",
-    "async for event in graph.astream_events(input, version=\"v2\"):\n",
-    "    if event[\"event\"] == \"on_chat_model_end\":\n",
-    "        print(event[\"data\"][\"output\"].content, end=\"\", flush=True)"
+    "try:\n",
+    "    async for event in graph.astream_events(input, version=\"v2\"):\n",
+    "        if event[\"event\"] == \"on_chat_model_end\":\n",
+    "            print(event[\"data\"][\"output\"].content, end=\"\", flush=True)\n",
+    "except:\n",
```

Comment:
why is this needed?

Reply:
because it throws an error - on purpose, but the CI doesn't know that. All errors that we throw on purpose to show something failing (usually we show the way to fix it after) need to be wrapped in try/excepts for the CI

Reply:
cool, makes sense